model/flow/quorum_certificate.go

```diff
@@ -25,3 +25,13 @@ type QuorumCertificate struct {
 	// from all signers.
 	SigData []byte
 }
+
+// QuorumCertificateWithSignerIDs is a QuorumCertificate model with signer IDs instead of indices.
+// it is needed for backward-compatibility with the FlowEpoch smart contract which generates QCs
+// using signer IDs.
+type QuorumCertificateWithSignerIDs struct {
```

Comment:
To me, it would make sense to move this struct definition somewhere to the bootstrapping folder. This would highlight that this data structure is not used to for building blocks and needs explicit post-processing (as much of the bootstrapping data)

Reply:
Yeah, makes sense, will address in later PR